apps/governance/src/lib/tranches/tranches-store.tsx

```diff
@@ -50,7 +50,16 @@ export const useTranches = create<TranchesStore>()((set) => ({
         ?.map((t) => {
           const tranche_progress =
             t.duration !== 0 ? (now - t.cliff_start) / t.duration : 0;
-          const lockedDecimal = tranche_progress < 0 ? 1 : 1 - tranche_progress;
+          const lockedDecimal =
+            t.duration !== 0
+              ? tranche_progress < 0
+                ? 1
+                : 1 - tranche_progress
+              : now < t.cliff_start
+              ? 1
+              : 0;
```

Comment:
Sorry to be annoying but can we make this an if statement or a function or something, this is hard to read.